website/static/js/dateRangePicker.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Date range picker.
+ */
+
+'use strict';
+
+require('pikaday-css');
+
+var oop = require('js/oop');
+var pikaday = require('pikaday');
+
+
+var DateRangePicker = oop.defclass({
+    constructor: function(startPickerElem, startDate, endPickerElem, endDate, minDate, maxDate) {
```

Comment:
Minor: This is ok for now, but with that many arguments, it's more readable to use an options object to pass arguments:

```
new DateRangePicker({
    startPicker: '#startPicker',
    startDate: new Date(...),
     //...
});
```